### PR TITLE
[NR-24565] Android Agent 6.9.0 release notes & new app launch time doc

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - new feature doc, android agent 6.9.0 release notes

Android agent is going to support for reporting app launch time (cold time and hot time only).
Nav: Mobile monitoring --> android agent --> install and config section

Along with a new android release 6.9.0

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

**Review by zuluecho9 — DISMISSED** [review]

**Comment by ywang-nr** [comment]
Target release date: 9/27 in the afternoon

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Configure app launch time [#app-launch-time]
+
+<Callout variant="important">
+This feature is available for agent versions 6.9.0 and higher.
+</Callout>
+
+To enable the Android agent to report app launch time: Add the following to your application's `AndroidManifest.xml`:
+   ```
+   <provider
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx:22]: yes, will do.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-690.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+subject: Android agent
+releaseDate: '2022-09-21'
+version: 6.9.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.9.0.zip'
+---
+
+## New in this release
+* Add support for reporting app launch time, including cold time and hot time
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-690.mdx:9]: okay, will do

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/android-agent-configuration-feature-flags.mdx [hunk 3]
```diff
@@ -637,3 +637,42 @@ All settings, including the call to invoke the agent, are called in the `onCreat
...
 </CollapserGroup>
+
+## App launch time settings [#applaunchtime]
+
+<CollapserGroup>
+  <Collapser
+    id="ff-applaunchtime"
+    title="FeatureFlag.AppStartMetrics"
+  >
+    Enable or disable reporting app launch time (cold and hot time) to the [MobileRequest](/docs/insights/nrql-new-relic-query-language/nrql-query-examples/insights-query-examples-new-relic-mobile#mobilerequest-examples) event type.
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/android-agent-configuration-feature-flags.mdx]: corrected to metrics and add more details for the definition

**Review by cthomas-newrelic — APPROVED** [review]
LGTM! Please have docs folks go over it for clarity

**Comment by ywang-nr** [comment]
Ready for merge and release on 9/28(today)

**Review by homelessbirds — APPROVED** [review]

**Comment by homelessbirds** [comment]
This looks great-- thank you!

**Pull request merged**